COLLABORATOR_GUIDE.md

```diff
@@ -64,7 +64,8 @@ from other Collaborators. Leave at least 48 hours during the week and
 72 hours over weekends to account for international time differences
 and work schedules. Trivial changes (e.g. those which fix minor bugs
 or improve performance without affecting API or causing other
-wide-reaching impact) may be landed after a shorter delay.
+wide-reaching impact), and changes that affect only specific parts of
```

Comment:
Nit, non-blocking: s/only specific/small/

_specific_ may cause someone to ask "And which specific parts are those?"

(Of course "small parts" may cause someone to ask "What is small?" but the answer there is "We trust collaborator's judgment to know when a change is small or not."

If "small" is not the right word here, maybe "focused" or "localized"? (Although "localized" runs the risk of being confused with language localization.)

Reply:
Maybe "small changes that affect only documentation and/or the test suite" would work?

Reply:
I actually like “focused” pretty much – e.g. a PR that refactors `var` to `let`/`const` doesn’t need to be small but can still be focused